src/Umbraco.Web.UI.Client/src/common/directives/components/umbicon.directive.js

```diff
@@ -15,16 +15,7 @@ Simple icon
 
 Icon with additional attribute. It can be treated like any other dom element
 <pre>
-    <umb-icon icon="icon-alert" class="icon-class" ng-click="doSomething()"></umb-icon>
-</pre>
-
-Manual svg string
-This format is only used in the iconpicker.html
-<pre>
-    <umb-icon 
-        icon="icon-alert"
-        svg-string='<svg height="50" width="50"><circle cx="25" cy="25" r="25" fill="red" /></svg>'>
-    </umb-icon>
```

Comment:
So does this notation no longer work? Technically that's already a breaking change! 😅

Reply:
The notation will still work, but it's isn't being used anywhere in the umbraco backoffice code anymore. 

When you define a value for the `icon` attribute in the `umb-icon` directive, it will make a request looking for that icon. If it finds one it will return an svg string and will assign the value of that string to the `svg-string` attribute, which the directive uses internally to render out the svg. So the attribute is used by the directive anyway, but it just doesn't really make sense to set it directly anymore.